wallet/wallet.go

```diff
@@ -288,44 +281,10 @@ func (w *Wallet) GetVoteBitsExtended() []byte {
 
 // SetTicketPurchasingEnabled is used to enable or disable ticket purchasing in the
 // wallet.
-func (w *Wallet) SetTicketPurchasingEnabled(flag bool) error {
+func (w *Wallet) SetTicketPurchasingEnabled(flag bool) {
```

Comment:
That's the only way the RPC server to know whether or not purchasing is actually happening IIRC. I have it being called from start/stop ticketbuyer.